models/utility_functions.py

```diff
@@ -50,6 +50,8 @@ def float_to_bits(x):
 
 
 def torch_random(shape, low, high, dtype):
+    if dtype == torch.int64:
```

Comment:
should we use is_floating_point here to check?

Reply:
Good idea! I will do that

Reply:
Unfortunately, it only works on tensors. Not on dtype objects

Reply:
It should work for dtypes, it's a property
<img width="505" alt="image" src="https://github.com/tenstorrent-metal/tt-metal/assets/109362939/12d5c23e-f6d1-43b6-b1ea-2c162e8300f2">